Spine/DeserializeOperation.swift

```diff
@@ -89,7 +89,11 @@ class DeserializeOperation: Operation {
 
 			if let data = data["included"].array {
 				for representation in data {
-					try extractedIncludedResources.append(deserializeSingleRepresentation(representation))
+                    do {
+                        try extractedIncludedResources.append(deserializeSingleRepresentation(representation))
+                    } catch SerializerError.resourceTypeUnregistered(let resourceType) {
+                        Spine.logInfo(.serializing, "Cannot perform deserialization for resource type '\(resourceType)' because it is not registered.")
```

Comment:
Sure, updated to logWarning - any idea why this breaks travis build for tvOS? https://travis-ci.org/wvteijlingen/Spine/builds/171061035